symfony/client/src/orangehrmAdminPlugin/pages/systemUser/SystemUser.vue

```diff
@@ -213,6 +216,7 @@ export default {
   },
 
   setup() {
+    const filters = ref({...defaultFilters});
```

Comment:
Are we going to totally depend on composition API? can't we define under data :thinking:

Reply:
we cannot as `setup()` is called before vue options api is invoked.